pimpmycause/pimpmycause/templates/profiles/profile.html

```diff
@@ -8,109 +8,100 @@
+
+                                {% else %}
+                                    <img class="profile-img" src="{{profile_update_form.image.value}}" alt="{{ user.name }} {{ user.surname }} Profile Image">
+                                {% endif %}
+
+                                    {{ profile_update_form.media }}
+                                    {{ profile_update_form.image }}
+                            </div>
+                            <div class="large-offset-3 medium-offset-2 large-4 medium-6 small-12 small-centered medium-uncentered columns">
+                                {{ profile_update_form.name.errors }}
+                                <label for="name"> Name </label>
+                                <input type="text" name="name" placeholder="Sam" value='{{ profile_update_form.name.value|default_if_none:"" }}' required>
 
-                                        {{ profile_update_form.position.errors }}
-                                        <label for="position"> Position </label>
-                                        <input type="text" name="position" placeholder="Marketer" value='{{ profile_update_form.position.value|default_if_none:"" }}' required>
+                                {{ profile_update_form.surname.errors }}
+                                <label for="surname"> Surname </label>
+                                <input type="text" name="surname" placeholder="Jones" value='{{ profile_update_form.surname.value|default_if_none:"" }}' required>
 
-                                        {{ profile_update_form.city.errors }}
-                                        <label for="city"> City </label>
-                                        <input type="text" name="city" placeholder="London" value='{{ profile_update_form.city.value|default_if_none:"" }}' required>
+                                {{ profile_update_form.position.errors }}
+                                <label for="position"> Position </label>
+                                <input type="text" name="position" placeholder="Marketer" value='{{ profile_update_form.position.value|default_if_none:"" }}' required>
 
-                                        {{ profile_update_form.country.errors }}
-                                        {{ profile_update_form.country }}
+                                {{ profile_update_form.city.errors }}
+                                <label for="city"> City </label>
+                                <input type="text" name="city" placeholder="London" value='{{ profile_update_form.city.value|default_if_none:"" }}' required>
 
-                                    </div>
-                                </div>
-                            </div>
-                            <div class="row">
-                                <h2>Contact Info &amp; Social</h2>
+                                {{ profile_update_form.country.errors }}
```

Comment:
maybe we could add a label for country too ?
well unless if you find it unnecessary ..